### PR TITLE
ARTEMIS-1838 Fixing AMQP->Core Race on durable property

### PR DESCRIPTION
**Pull request by clebertsuconic**

### CODE REVIEW

File: tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/crossprotocol/MultiThreadConvertTest.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+import org.slf4j.LoggerFactory;
+
+public class MultiThreadConvertTest extends SmokeTestBase {
+
+   private static final String SERVER_NAME_0 = "standard";
+
+   private static final Logger LOG = LoggerFactory.getLogger(MultiThreadConvertTest.class);
+
+   protected String queueName = "amqTestQueue1";
+   private SimpleString coreQueue;
```
Comment (tabish121) [inline tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/crossprotocol/MultiThreadConvertTest.java]: This coreQueue variable is never used and should be removed.

**Comment by tabish121** [comment]
Now you have a checkstyle violation :)

**Comment by clebertsuconic** [comment]
Sh...t :)

**Comment by clebertsuconic** [comment]
@tabish121 fixed again..

**Comment by tabish121** [comment]
@clebertsuconic ok, I was about to fix it myself.  I confirmed that the fix does seem to resolve the issue, I can no longer reproduce it here using the test and an external broker or with the smoke test.

**Pull request merged**